### DISCUSSION TITLE
Environment variables doesn't seems to have effects

### DISCUSSION THREAD

**Discussion by SebDominguez** [inventree/InvenTree#6150] [Q&A]
Following the [Docker production server installation guide](https://docs.inventree.org/en/latest/start/docker_prod/#docker-production-server) 

After trying to edit the `.env` file by setting `INVENTREE_COMPANY_NAME` or `INVENTREE_EMAIL_PREFIX` or even `INVENTREE_BASE_URL` doesn't have any effect on the Inventree configuration.

Company name stay blank, email subjects are still prefixed with [Inventree] no matter what I put on the `INVENTREE_EMAIL_PREFIX` environment variable, and the Base URL stay blank.

Are those settings only configurable via the `config.yaml` file?

**Comment by SchrodingersGat** [comment]
These should be taking precedence if set in the environment variables. Can you please check if any other of your environment variables are not working at expected?

**Comment by SebDominguez** [comment]
I've also tried `INVENTREE_TIMEZONE=Europe/Madrid`or `INVENTREE_TIMEZONE=ES` but the `docker compose logs` still show me the UTC timestamp. Still playing around so I didn't tried all the environment variable yet.

Just tried to change `INVENTREE_BASE_CURRENCY=EUR` to `INVENTREE_BASE_CURRENCY=USD` and after a `docker compose down && docker compose up -d` no changes where made.

I still need to change the language in GUI because the environment variable `INVENTREE_LANGUAGE` doesn't have any effects.

 Also came across weird things like the French translation having a lot of `\'` everywhere. 

<img width="1379" alt="Capture d’écran 2024-01-03 à 13 07 02" src="https://github.com/inventree/InvenTree/assets/52816569/febc5be0-242a-4512-9ba5-ab7eb69575e2">

Is there some step that I am missing in order to configure InvenTree ?

**Reply by SchrodingersGat** [reply]
@SebDominguez this may be due to how env variables are cached. Can you try:

1. `docker compose restart`
2. `docker compose stop && docker compose up -d`

And see if the environment variables are then loaded correctly in either case
